keps/sig-api-machinery/3488-cel-admission-control/README.md

```diff
@@ -2312,6 +2320,9 @@ in back-to-back releases.
   If multiple admission policies require the same conversion, convert only once. 
   From @liggitt: "webhook code loops up one level, first accumulates all the validation webhooks we'll run, then converts to the versions needed by those webhooks then evaluates in parallel"
 - authz check to the specific resource referenced in the policy's paramKind. ([comment](https://github.com/kubernetes/kubernetes/pull/113314#discussion_r1013135860))
+- complete feature of access to namespace metadata 
+- complete type check for CRD
+- add controlled rollout strategy to support future CEL library/function/variable changes
```

Comment:
Quantity support +tests?  I'm not confident quantity fields even work in a reasonable today (I hope so, but I don't have any data to support that).

Reply:
Added